public/style.css

```diff
@@ -10,12 +11,38 @@ label {
   display: block;
 }
 
-nav a {
-  display: inline-block;
-  margin: 1em;
+.Auth-form {
```

Comment:
TIL of the naming convention that block elements begin with a capital letter.

Reply:
I'd personally recommend sticking with `kebab-case` – no need for capitalization and capitals often muck things up in other contexts. YMMV. https://benfrain.com/modular-css-bem-oocss-naming/